dms/models/dms_file.py

```diff
@@ -98,48 +90,63 @@ class File(models.Model):
         required=True,
         store=False,
     )
-
     extension = fields.Char(compute="_compute_extension", readonly=True, store=True)
-
     mimetype = fields.Char(
-        compute="_compute_mimetype", string="Type", readonly=True, store=True
+        string="Type", compute="_compute_mimetype", readonly=True, store=True
     )
-
     size = fields.Float(readonly=True)
     human_size = fields.Char(
-        readonly=True,
         string="Size (human readable)",
+        readonly=True,
         compute="_compute_human_size",
         store=True,
     )
-
     checksum = fields.Char(string="Checksum/SHA1", readonly=True, index="btree")
-
-    content_binary = fields.Binary(attachment=False, prefetch=False, invisible=True)
-
+    content_binary = fields.Binary(attachment=False, prefetch=False)
     save_type = fields.Char(
-        compute="_compute_save_type",
         string="Current Save Type",
-        invisible=True,
+        compute="_compute_save_type",
         prefetch=False,
     )
-
     migration = fields.Char(
-        compute="_compute_migration",
         string="Migration Status",
+        compute="_compute_migration",
         readonly=True,
         prefetch=False,
         compute_sudo=True,
+        store=True,
```

Comment:
Why doing it stored? I don't think it's needed and it can lead to side effects if the depends is not correct.